IPython/core/magics/script.py

```diff
@@ -208,15 +209,23 @@ def in_thread(coro):
             """Call a coroutine on the asyncio thread"""
             return asyncio.run_coroutine_threadsafe(coro, event_loop).result()
 
+        async def _readchunk(stream):
+            try:
+                return await stream.readuntil(b"\n")
```

Comment:
\n is by default but i think its more obvious what we're doing here by just specifying it. though not hard opinion